src/main/java/org/broadinstitute/consent/http/authentication/AuthorizationHelper.java

```diff
@@ -49,6 +49,13 @@ protected AuthUser buildAuthUserFromHeaders(Map<String, String> headers) {
       logWarn(String.format(
           "Reading oauth2 claim headers: email is null, auth user is incomplete. Aud: %s Name: %s",
           aud, name));
+    } else {
+      try {
```

Comment:
If we've got an email address for the user, look them up and run the enforcement logic.

Needs to be here because of PermitAll in our code.  

Future refactoring might include replacing DuosUser by adding an Optional of the User to AuthUser so that it can leverage the obtained User object to be used in existing endpoints.